src/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -71,6 +73,47 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     }
   }
 
+  renderGoogleAdSnippet() {
+    const { artwork, googleAdId: fromPropsGoogleAdId } = this.props
+    const { GOOGLE_ADWORDS_ID: fromSharifyGoogleAdId } = sd
+    const { is_in_auction, is_acquireable, _id } = artwork
+    if (!is_in_auction && !is_acquireable) return
+
+    // TODO: Investigate always being able to select from sharify.
```

Comment:
This dance is b/c during the SSR render, `sd.GOOGLE_ADWORDS_ID` is undefined. So we wind up passing that in to the app context from Force  via `res.locals.sd.GOOGLE_ADWORDS_ID`.

I think this is what we have to do to be able to consistently access that value properly on both the server and the client (this follows a pattern we've used elsewhere).

This should be safely mergeable in the meantime, and there's an accompanying small Force PR.